dspace-api/src/main/resources/org/dspace/storage/rdbms/sqlmigration/h2/V1.2__Initial_DSpace_1.2_Oracle_database_schema.sql

```diff
@@ -102,9 +102,9 @@ CREATE TABLE EPerson
   password            VARCHAR2(64),
   firstname           VARCHAR2(64),
   lastname            VARCHAR2(64),
-  can_log_in          NUMBER(1),
-  require_certificate NUMBER(1),
-  self_registered     NUMBER(1),
+  can_log_in          BOOLEAN,
+  require_certificate BOOLEAN,
+  self_registered     BOOLEAN,
```

Comment:
Don't we need a new Flyway migration to handle all these type changes?  I'm slightly worried about directly modifying all these old migration files.. existing databases will not realize these old files were updated (Flyway never re-runs old migrations). Therefore in existing databases, fields like "can_log_in" will remain NUMBER(1) instead of BOOLEAN.